openreview/conference/invitation.py

```diff
@@ -801,17 +788,97 @@ def set_registration_invitation(self, conference, start_date, due_date, with_are
             replyto = None,
             forum = None,
             content = {
-                'title': registration_parent_invitation.reply['content']['title']['value'],
-                'subject_areas': registration_parent_invitation.reply['content']['subject_areas']['value'],
-                'profile confirmed': registration_parent_invitation.reply['content']['profile confirmed']['value'],
-                'TPMS account confirmed': registration_parent_invitation.reply['content']['TPMS account confirmed']['value'],
+                "Instructions": "Help us get to know our reviewers better and the ways to make the reviewing process smoother by answering these questions. If you don't see the questionnaire form below, click on the blue \"Registration\" button.",
+                "title": "Questionnaire for Reviewers"
             }
         ))
 
+        registration_content = {
+            'title': {
+                'value': conference.get_short_name() + ' Registration',
+                'order': 1
+            },
+            'profile confirmed': {
```

Comment:
I'm not vey sure about having two fields to confirm profile and expertise. I think they are the same but @mohituniyal  is right saying there are two different UIs to check this, @mspector what is your opinion?